app/dashboard/components/dashboard-section.njk

```diff
@@ -11,17 +11,17 @@
         {% endif %}
       </span>
       </div>
-      <div class="nhsuk-grid-column-two-quarters">
+      <div>
       {% if section.isActive %}
-        <span data-test-id="dashboard-section-requirement" class="app-section-list__task-name">
-          <p>{{ section.requirement }}</p>
+        <span data-test-id="dashboard-section-requirement" class="bc-u-block-float-left nhsuk-u-margin-left-4">
+          <p class="nhsuk-u-margin-bottom-0">{{ section.requirement }}</p>
         </span>
-        <strong data-test-id="dashboard-section-status" class="govuk-tag app-section-list__task-completed" id="eligibility-completed">
+        <strong data-test-id="dashboard-section-status" class="bc-c-section-list__task-tag" id="eligibility-completed">
```

Comment:
I am wondering if we can use our tag component here?

Reply:
Also this is probably something I've put in but what is `id="eligibility-completed"` about? can we get rid of it of we don't need it

Reply:
we can use tag component (not this ticket!) also I didn't add it but can remove it

Reply:
cool yeah let's do that in a separate ticket and lets remove that id. 

I copied and pasted these from the prototypes without much thought so my bad on that one.